src/flows/pipes/RawFluxEditor/view.tsx

```diff
@@ -76,10 +76,15 @@ const Query: FC<PipeProp> = ({Context}) => {
   const injectIntoEditor = useCallback(
     (fn): void => {
       let text = ''
-      if (fn.name === 'from' || fn.name === 'union') {
-        text = `${fn.example}`
+      if (isFlagEnabled('fluxDynamicDocs')) {
```

Comment:
Is `CLOUD` already guaranteed here? Maybe we should be explicit and check to make sure we in `CLOUD`

Reply:
we explicitly check if we're in `CLOUD` in `TimeMachine` component, so it would make sense to include it here as well